compiler/rustc_data_structures/src/thin_vec.rs

```diff
@@ -20,6 +20,13 @@ impl<T> ThinVec<T> {
     pub fn iter_mut(&mut self) -> std::slice::IterMut<'_, T> {
         self.into_iter()
     }
+
+    pub fn push(&mut self, item: T) {
```

Comment:
Hmm, in an old version of this PR, you used `.extend([...])` with one element, so I requested adding `fn push()`. But now that it's added, I don't see any (new) uses of either `extend` or `push` in rustdoc. Do we still need `push`? Am I missing something?

Reply:
The old code called push on a Vec. I added a ThinVec, but it didn't have push, so I changed the push calls to extend.

But then I added the push method to ThinVec, so I reverted the method call changes. There is no diff for these now, since the code is the same - push is called.

Reply:
Ah, of course! Thanks :)